flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/flink/FlinkContainer.java

```diff
@@ -281,21 +280,19 @@ public FlinkContainer build() {
 					numTaskManagers,
 					logBackupDirectory.orElse(null));
 			} catch (Exception e) {
-				throw new RuntimeException("Could not build the flink-dist image", e);
-			} finally {
 				temporaryFolder.delete();
+				throw new RuntimeException("Could not build the flink-dist image", e);
 			}
 		}
 
 		private ImageFromDockerfile buildConfiguredImage(
-				String flinkDistName,
 				Path workersFile,
 				String baseImage) {
 			return new ImageFromDockerfile(
 				"flink-dist-configured")
 				.withDockerfileFromBuilder(
 					builder -> builder.from(baseImage)
-						.copy("workers", flinkDistName + "/conf/workers")
+						.copy("workers", "flink/conf/workers")
```

Comment:
Here we are saying "copy the 'workers' file to the 'flink/conf/workers'" of the new image, right? What is in the 'workers' file? Shouldn't this be `/conf/workers`? Also on the master the args are reversed.

Reply:
Yes, the version on the master is wrong.

The workers file is one created manually above. See line 254.

Reply:
Ok after an offline discussion, this comment is not relevant anymore.